### PR TITLE
ADD default locale parameter

### LINKED ISSUE #1: Default locale should be configurable.

### PR DESCRIPTION
**Pull request by labajo**
closes #1

### CODE REVIEW

File: lib/l10n.js [hunk 1]
```diff
@@ -8,13 +8,19 @@ var i18n = require('i18n');
...
+module.exports = function(locales, directory, defaultLocale) {
+  var configuration = {
     updateFiles: false,
     locales: locales,
     directory: directory || __dirname,
     objectNotation: true
-  });
+  };
+
+  if (defaultLocale) {
```
Comment (jlorgal) [inline lib/l10n.js]: do you really need this condition? Is it not enough to assign the value?

File: test/unit/themible-test.js [hunk 2]
```diff
@@ -59,7 +60,7 @@ describe('Themible Middleware Tests', function() {
     var exception = false;
     try {
       themible(appMock, path.join(__dirname, 'testThemes',
-          'themeNoLocales'), '/themePath');
+          'themeNoLocales'), '/themePath', 'en');
```
Comment (jlorgal) [inline test/unit/themible-test.js:63]: I haven't seen any test without a default locale.

Comment (labajo) [inline test/unit/themible-test.js:63]: They are included in the l10n module tests.

**Comment by jlorgal** [comment]
LGTM

**Pull request merged**